frontend/src/lib/components/Cards/InsightCard/InsightCard.tsx

```diff
@@ -513,7 +513,11 @@ export function InsightViz({
             ) : empty ? (
                 <InsightEmptyState />
             ) : timedOut ? (
-                <InsightTimeoutState isLoading={!!loading} insightProps={{}} insightType={insight.filters.insight} />
+                <InsightTimeoutState
+                    isLoading={!!loading}
+                    insightProps={{ dashboardItemId: insight.short_id }}
```

Comment:
just followed the pattern we do elsewhere. There might be something else more meaningful here